### PR TITLE
[ADD] Add company types data module

### PR DESCRIPTION
**Pull request by lfreeke**
Add company types data. Depends on partner_company_type module.

### CODE REVIEW

**Comment by lfreeke** [comment]
@NL66278 ping

File: l10n_nl_company_type/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
```
Comment (NL66278) [inline l10n_nl_company_type/__manifest__.py]: https everywhere (actually can also be done for License, but do not know wether that is according to oca standards)

File: l10n_nl_company_type/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "l10n_nl_company_type",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Dutch localization",
+    "summary": "",
```
Comment (NL66278) [inline l10n_nl_company_type/__manifest__.py]: Please add a summary

**Review by NL66278 — APPROVED** [review]
LGTM and test

File: l10n_nl_company_type/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+l10n_nl_company_type
```
Comment (hbrunn) [inline l10n_nl_company_type/README.rst]: use a human readable name

File: l10n_nl_company_type/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+====================
+
+This module was written to extend the functionality of Dutch company types.
+The module depens on partner_company_type
```
Comment (hbrunn) [inline l10n_nl_company_type/README.rst]: depenDs

File: l10n_nl_company_type/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+Usage
+=====
+
+#. go to ...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/{repo_id}/10.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (hbrunn) [inline l10n_nl_company_type/README.rst]: resolve repo_id

File: l10n_nl_company_type/README.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Known issues / Roadmap
+======================
+
+* ...
```
Comment (hbrunn) [inline l10n_nl_company_type/README.rst]: remove empty section

File: l10n_nl_company_type/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "l10n_nl_company_type",
```
Comment (hbrunn) [inline l10n_nl_company_type/__manifest__.py]: friendly name

File: l10n_nl_company_type/data/l10n_nl_company_types.xml [hunk 8]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="company_type_stichting" model="res.partner.company.type">
+            <field name="name">Stichting</field>
```
Comment (hbrunn) [inline l10n_nl_company_type/data/l10n_nl_company_types.xml]: this is a translatable field, so add the data in English and add a translation file

File: l10n_nl_company_type/data/l10n_nl_company_types.xml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+            <field name="shortcut"></field>
+        </record>
+        <record id="company_type_eenmanszaak" model="res.partner.company.type">
+            <field name="name">Eenmanszaak</field>
+            <field name="shortcut">ez</field>
```
Comment (hbrunn) [inline l10n_nl_company_type/data/l10n_nl_company_types.xml]: same here

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Comment by lfreeke** [comment]
@hbrunn fixed

File: l10n_nl_dutch_company_type/data/l10n_nl_dutch_company_types.xml [hunk 10]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/data/l10n_nl_dutch_company_types.xml]: You have the same file twice, one with plural one with singular name. Use git rm

**Review by hbrunn — APPROVED** [review]

**Pull request merged**